modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/resources/OSGI-INF/bulk-contrib.xml

```diff
@@ -6,6 +6,9 @@
   <extension target="org.nuxeo.ecm.core.bulk" point="actions">
     <action name="automation" inputStream="bulk/automation" bucketSize="100" batchSize="10" httpEnabled="true"
       validationClass="org.nuxeo.ecm.automation.core.operations.services.bulk.validation.AutomationBulkValidation" />
+
+    <action name="automation-ui" inputStream="bulk/automationUi" bucketSize="100" batchSize="10" httpEnabled="true"
```

Comment:
In the continuity of @ataillefer comment, what do you think about renaming `automation-ui` to `automationUi` in order to follow what was done for previous actions?

This will allow us to use the constant `ACTION_NAME` to build the bulk command in java code.

Reply:
ok let's rename the action to automationUi then